iis/handle.go

```diff
@@ -76,8 +76,34 @@ func (h *taskHandle) run(driverConfig *TaskConfig) {
 		h.logger.Error("unable to initialize stats", "error", err)
 	}
 
+	websiteConfig := WebsiteConfig{
+		Name: h.taskConfig.AllocID,
+		Env:  map[string]string{},
+		AppPoolIdentity: iisAppPoolIdentity{
+			Identity: driverConfig.AppPoolIdentity,
+		},
+		AppPoolConfigPath: driverConfig.AppPoolConfigPath,
+		SiteConfigPath:    driverConfig.SiteConfigPath,
+	}
+
+	// Setup environment variables.
+	// NOMAD_APPPOOL_* are keywords for applying user/pass info for a given Application Pool in a secure manner
+	for key, val := range h.taskConfig.Env {
+		switch key {
+		case "NOMAD_APPPOOL_USERNAME":
+			websiteConfig.AppPoolIdentity.Identity = "SpecificUser"
+			websiteConfig.AppPoolIdentity.Username = val
+			break
```

Comment:
In golang, `switch` cases are not fall through. `break` is not needed.

https://play.golang.org/p/SEGfaIDWCC-